mysql-test/suite/rpl/t/bug84415.test

```diff
@@ -0,0 +1,265 @@
+-- source include/master-slave.inc
```

Comment:
The host executing test might take arbitrarily long time to execute any piece of code due to load. There is no guarantee that something that has to executed every 300ms will be executed at least once in one second, it is only known for sure that it will not be executed more often than that, and that's all.